source-install.yml

```diff
@@ -20,3 +25,4 @@
     - pulp3-workers
     - pulp3-resource-manager
     - pulp3-webserver
+    - pulp3-devel
```

Comment:
Does pulp3-devel work without the previous roles? If not, perhaps we should remove them and put them in the pulp3-devel roles meta dependencies.